packages/api/src/middleware/validation/assignTask.js

```diff
@@ -25,6 +25,11 @@ async function validateAssigneeId(req, res, next) {
     return res.status(403).send('Not authorized to assign other people for this task');
   }
 
+  // if the assignee_user_id is null, this means that the task is 'Unassigned' which is a valid
```

Comment:
I think it's interesting that line 24 just above acknowledges the value could be `null`.